Source/WebCore/editing/ReplaceNodeWithSpanCommand.cpp

```diff
@@ -69,9 +69,10 @@ void ReplaceNodeWithSpanCommand::doApply()
 
 void ReplaceNodeWithSpanCommand::doUnapply()
 {
-    if (!m_spanElement || !m_spanElement->isConnected())
+    RefPtr spanElement = m_spanElement;
+    if (!spanElement || !spanElement->isConnected())
         return;
-    swapInNodePreservingAttributesAndChildren(m_elementToReplace, *m_spanElement);
+    swapInNodePreservingAttributesAndChildren(m_elementToReplace.copyRef(), *spanElement);
```

Comment:
Do we really need to write `copyRef` here?

Reply:
Even though `m_elementToReplace` is a smart pointer, it is not a local variable and thus may go away during the execution of the function. As a result, the static analyzer was complaining. The copyRef() makes sure we have a *local* smart pointer guaranteed to live during the execution of the function.

Reply:
May look nicer to have swapInNodePreservingAttributesAndChildren() take in a `Ref<HTMLElement>`.